vg/runtime/wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Just, FYI, these will go away. @jvivian is working on this. John, we should bite the bullet and remove all of them at once, don't you think?